inference-engine/tests/functional/inference_engine/ir_serialization/tensor_iterator.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <fstream>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include "gtest/gtest.h"
+#include "ie_core.hpp"
+#include "ie_blob.h"
+#include "common_test_utils/data_utils.hpp"
+
+#ifndef IR_SERIALIZATION_MODELS_PATH  // should be already defined by cmake
+#define IR_SERIALIZATION_MODELS_PATH ""
+#endif
+
+class SerializationTensorIteratorTest : public ::testing::Test {
+protected:
+    std::string test_name =
+        ::testing::UnitTest::GetInstance()->current_test_info()->name();
+    std::string m_out_xml_path = test_name + ".xml";
+    std::string m_out_bin_path = test_name + ".bin";
+
+    void TearDown() override {
+        std::remove(m_out_xml_path.c_str());
+        std::remove(m_out_xml_path.c_str());
+    }
+
+    void serialize_and_compare(const std::string& model_path, InferenceEngine::Blob::Ptr weights) {
+        std::stringstream buffer;
+        InferenceEngine::Core ie;
+
+        std::ifstream model(model_path);
+        buffer << model.rdbuf();
```

Comment:
Can be add `ASSERT_TRUE(model);` here for test the test?

Reply:
Assert is added.